docs/tutorials/blob-url.md

```diff
@@ -0,0 +1,28 @@
+# Blob URL
+
+In case you need shaka-player to load a blob url mpd:
+
+#### For version v2.x
+
+- Register blob scheme to network engine (note: you don't have to do this in v3.x)
+  `shaka.net.NetworkingEngine.registerScheme('blob', shaka.net.HttpFetchPlugin);`
+- With blobs, neither extension nor MIME type can be used to deduce which manifest parser to use.
+  But there is an easy fix is register manifestParserFactory when load player
+  `player.load: player.load(uri, null, shaka.dash.DashParser);`
```

Comment:
Not sure why you have `player.load: ` before the call.  Would you please remove it?